src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -87,6 +111,11 @@ public abstract class RemoteAuthenticationHandler<TOptions> : AuthenticationHand
             return true;
         }
 
+        /// <summary>
+        /// Authenticate the user identity with the identity provider. 
+        ///
+        /// This could be done through a back channel communication with the identity provider. 
```

Comment:
The backchannel isn't the important part here, it's that this processes requests on the CallbackPath endpoint.